token-holders/how-to-stake.md

```diff
@@ -0,0 +1,40 @@
+{% hint style="info" %}
```

Comment:
seems like adding the title or not doesn't really matter as it shows on the page any way but maybe we want to standardize -- either we omit all the main titles or show them all wdyt? for ref: line 1 how how-to-vote.md